BraveWallet/Crypto/Transactions/TransactionDetailsView.swift

```diff
@@ -0,0 +1,264 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveCore
+import BraveUI
+import SwiftUI
+import Swift
+import struct Shared.Strings
+
+struct TransactionDetailsView: View {
+  
+  var info: BraveWallet.TransactionInfo
+  @ObservedObject var networkStore: NetworkStore
+  @ObservedObject var keyringStore: KeyringStore
+  var visibleTokens: [BraveWallet.BlockchainToken]
+  var assetRatios: [String: Double]
+  
+  @Environment(\.presentationMode) @Binding private var presentationMode
+  @Environment(\.openWalletURLAction) private var openWalletURL
+  
+  private let dateFormatter = DateFormatter().then {
+    $0.dateFormat = "h:mm a - MMM d, yyyy"
+  }
+  
+  private let numberFormatter = NumberFormatter().then {
+    $0.numberStyle = .currency
+    $0.currencyCode = "USD"
+  }
+
+  /// The value of the transaction with symbol, ex. `"0.0100 ETH"`
+  private var value: String {
+    let formatter = WeiFormatter(decimalFormatStyle: .balance)
+    let amount: String
+    switch info.txType {
+    case .erc20Transfer:
+      amount = info.txArgs[0]
+    case .erc721TransferFrom, .erc721SafeTransferFrom:
+      amount = "1" // Can only send 1 erc721 at a time
```

Comment:
lol where did you get that?

Reply:
It was from desktop transaction parser:
https://github.com/brave/brave-core/blob/master/components/brave_wallet_ui/common/hooks/transaction-parser.ts#L276
But actually looks like I might need to display a different value for tx details:
https://github.com/brave/brave-core/blob/master/components/brave_wallet_ui/components/extension/transaction-detail-panel/index.tsx#L140